src/tiledb/cloud/client.py

```diff
@@ -107,6 +100,18 @@ def login(
             os.getenv("TILEDB_REST_IGNORE_SSL_VALIDATION", "False")
         )
 
+    if host is None:
+        host = config.default_host
+    # See sc-56351. Usually, a hostname doesn't include a protocol
+    # scheme, but our SDK strictly requires the http(s) scheme.
+    elif not host.startswith(("http://", "https://")):
+        if verify_ssl:
+          host = f"https://{host}"
+        else:
+          # Don't use https:// for non-SSL connections
+          # (defaulting to https:// when user sets verify_ssl=False can be very confusing!)
```

Comment:
`verify_ssl=False` is useful when you have certificates that are not recognized by the client, such as self signed certs. This variable does not mean "no ssl".